src/test/java/by/exadel/internship/formTest/FormTest.java

```diff
@@ -66,7 +66,8 @@ public void givenFormWithFile_checkTestData()
         assertEquals(formFullDTO.getEmail(), "string");
         assertEquals(formFullDTO.getEnglishLevel(), EnglishLevel.A0);
         assertEquals(formFullDTO.getExperience(), "string");
-        assertEquals(formFullDTO.getFilePath(), "text.txt");
+        //wrong test - (will learn and use @Mock @MocKBean in next PR)
+        assertEquals(formFullDTO.getFilePath().contains("https://firebasestorage.googleapis.com/v0/b/gs://internship-project-e202a.appspot.com/o/"), true);
```

Comment:
I would disable this test at all not to mess storage with test files.

Reply:
done